cspell.yaml

```diff
@@ -121,3 +121,5 @@ words:
 - UX
 - devs
 - CLI
+- ifm
+- idx
```

Comment:
Please follow the [contributing guide for testing](https://github.com/apache/ozone-site/blob/HDDS-9225-website-v2/CONTRIBUTING.md#spelling). It tells you how to run cspell, which will only run it on markdown files so these updates which come from css and js files are not necessary.